lib/driver.js

```diff
@@ -158,7 +169,7 @@ class YouiEngineDriver extends BaseDriver {
     if (this.caps.platformName !== null) {
       let appPlatform = this.caps.platformName.toLowerCase();
 
-      if (['yimac', 'yitvos', 'bluesky'].includes(appPlatform)) {
+      if (['yimac', 'yitvos', 'bluesky', 'yilinux'].includes(appPlatform)) {
```

Comment:
It is fine to call it yilinux since it is a custom platform for our driver

Reply:
Thanks for confirming 🙏